integration/query_test.go

```diff
@@ -254,7 +254,21 @@ func TestQueryBadSortType(t *testing.T) {
 	for name, tc := range map[string]struct {
 		command bson.D
 		err     *mongo.CommandError
+		message string
```

Comment:
Let's call it `altMessage` if that's how we call the assertion parameter

Reply:
Fixed.